Xamarin.Forms.Core/Cells/Cell.cs

```diff
@@ -241,6 +248,11 @@ static void OnIsEnabledPropertyChanged(BindableObject bindable, object oldvalue,
 			(bindable as Cell).OnPropertyChanged("HasContextActions");
 		}
 
+		static void OnBackgroundColorPropertyChanged(BindableObject bindable, object oldvalue, object newvalue)
+		{
+			(bindable as Cell).OnPropertyChanged(nameof(BackgroundColor));
```

Comment:
use direct cast

Reply:
you don't need to fire the propertyChanged, it's done as part of the BO logic

Reply:
Removed it altogether